lib/systems/examples.nix

```diff
@@ -137,6 +137,19 @@ rec {
     libc = "newlib";
   };
 
+  esp8266 = {
+    config = "xtensa-lx106-elf";
+    libc = "newlib";
+  };
+  esp32 = {
+    config = "xtensa-esp32-elf";
+    libc = "newlib";
+  };
+  esp32s2 = {
+    config = "xtensa-esp32s2-elf";
```

Comment:
this looks like a very non-standard triple to me? Why not `esp32s2-unknown-none-elf`?